libmscore/style.cpp

```diff
@@ -962,6 +972,15 @@ static const StyleType styleTypes[] {
       { Sid::textLineFontStyle,             "textLineFontStyle",            int(FontStyle::Normal) },
       { Sid::textLineColor,                 "textLineColor",                QColor(0, 0, 0, 255) },
       { Sid::textLineTextAlign,             "textLineTextAlign",            QVariant::fromValue(Align::LEFT | Align::VCENTER) },
+      { Sid::textLineSystemFlag,            "textLineSystemFlag",           false },
+
+      { Sid::systemTextLineFontFace,              "systemTextLineFontFace",             "FreeSerif" },
```

Comment:
Perhaps it should be "Edwin" now

Reply:
It surely should

Reply:
Same font <code>FreeSerif</code> is used at more places here, see among others line 985.
So better to rebase to 3.x and change all <code>FreeSerif</code> into <code>Edwin</code>??

Reply:
Yes

Reply:
Done.